.pre-commit-config.yaml

```
@@ -1,51 +1,41 @@
 repos:
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.0.275
+    hooks:
+      - id: ruff
+        args:
+          - --fix
```

Let's be consistent with the other hooks:

```suggestion
        args: [--fix]
```

And as suggested by https://github.com/astral-sh/ruff-pre-commit, should we exit with an error when it fixes, so that fixed commits aren't committed in the pre-fix state? 

```suggestion
        args: [--fix, --exit-non-zero-on-fix]
```